CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_snapchat_ads_source v0.4.0
+PR [#13](https://github.com/fivetran/dbt_snapchat_ads_source/pull/13) applies the Ad Reporting V2 updates:
+## 🎉 Feature Enhancements 🎉
+- Adds the `ad_squad_hourly_report` and `campaign_hourly_report` source tables and accompanying staging models
+- Adds the field descriptions and grain uniqueness tests for the new models 
+- Adds additional fields to existing models 
+- Applies README standardization updates
+- Introduces the identifier variable for all source models
+- Casts all timestamp fields using dbt_utils.type_timestamp()
+- Introduces metric passthrough capability
+
+## 🚨 Breaking Changes 🚨
+- Changes `snapchat_schema` and `snapchat_database` variable names to `snapchat_ads_schema` and `snapchat_ads_database` 
+- Updates model names to prefix with `snapchat_ads`
```

Comment:
Small request to add breaking changes to the top. This way it is the first thing people see before upgrading.